frontend/.eslintrc.json

```diff
@@ -20,6 +20,10 @@
         "@typescript-eslint"
     ],
     "rules": {
+        "no-unused-vars": "off",
+        "@typescript-eslint/no-unused-vars": [
+          "error"
+        ],
```

Comment:
I'm just curious, what's the difference between these two rules, since you set them differently?

Reply:
Before turning off no unused vars completely, I'd suggest looking at [this](https://stackoverflow.com/questions/57802057/eslint-configuring-no-unused-vars-for-typescript) to see if that might fix it without disabling the rule

Reply:
This doesn't disable it, it actually just makes it work as expected (see image).
<img width="509" alt="Test Variable to Check No-Unused-Vars" src="https://user-images.githubusercontent.com/7987279/92987784-8e5f0200-f47a-11ea-81fa-15dcb83b6d97.png">
Based on [this Stack Overflow post](https://stackoverflow.com/a/61555310/7517518) and [this referenced Github post](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-unused-vars.md), it disables the base rules and sets the ESLint rule instead.